docker/dev/centos/README.md

```diff
@@ -0,0 +1,46 @@
+# How to use this Dockerfile
+
+This image is intended for **development** of Orion Context Broker. This will create an environment on which Orion can compile and be run based on CentOS 6.
+
+If you are only interested in **running** Orion Context Broker in a container this image is not what you are looking for. 
+
+## Build the image
+
+You only need to do this once.
+
+    docker build -t orion-dev:centos6 .
+    
+## Run the container
+
+This assumes that there is another docker running with name `mongodb`. If you don't, type this:
+
+    docker run -d --name mongodb mongo
+
```

Comment:
perhaps here we can add another pre-requisite indicating that code should be available under /path/to/orion/source and that the code can be obtained by git clone https://github.com/telefonicaid/fiware-orion.git